src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -602,8 +602,9 @@ private JComboBox<String> getComboBox() {
 		if (statusComboBox == null) {
 			statusComboBox = new JComboBox<String>();
 			statusComboBox.addItem("");
-			for (InventoryStatus currentStatus : InventoryStatus.values()) {
-				statusComboBox.addItem(MessageBundle.getMessage("angal.inventory." + currentStatus));
+			List<String> statusList = medicalInventoryManager.getStatusList();
+			for (String status : statusList) {
+				statusComboBox.addItem(MessageBundle.getMessage(status));
```

Comment:
Doesn't `getStatusList()` return a list of localized statuses?  Why another `getMessager()`?